Runtime/AvatarCreator/Scripts/UI/Elements/LoginElement.cs

```diff
@@ -39,14 +40,25 @@ public void SendVerificationCode()
             AuthManager.SendEmailCode(emailField.text);
         }
 
+        public void MergeCurrentUserToRpmAccount()
```

Comment:
Why not just have 1 function with a boolean? 
EG SetMergeCurrentUserToRpmAccont(bool mergeSession). 
The bool will be settable when calling the function from button events in the inspector